starkinfra/creditnotepreview/__creditnotepreview.py

```diff
@@ -0,0 +1,78 @@
+from starkcore.utils.subresource import SubResource
+from starkinfra.utils import rest
+from ..creditnote.invoice.__invoice import Invoice, _resource as _invoice_resource
+from starkcore.utils.api import from_api_json
+
+
+class CreditNotePreview(SubResource):
+    """# CreditNotePreview object
+    CreditNotePreview are used to preview CCB contracts between you and your customers with
+    selected table type.
+    When you initialize a CreditNotePreview, the entity will not be automatically
+    send in the Stark Infra API. The 'create' function sends the objects
+    to the Stark Infra API and returns the list of preview data.
+    ## Parameters (required):
+    - type [string]: The type of the table. ex: "sac", "price", "american", "bullet", "custom"
+    - nominal_amount [integer]: amount in cents transferred to the credit receiver, before deductions. ex: nominal_amount=11234 (= R$ 112.34)
+    - scheduled [datetime.date, datetime.datetime or string]: date of transfer execution. ex: scheduled=datetime(2020, 3, 10)
+    - tax_id [string]: credit receiver's tax ID (CPF or CNPJ). ex: "20.018.183/0001-80"
+    ## Parameters (conditionally required):
+    - nominal_interest [float]: yearly nominal interest rate of the credit note, in percentage. ex: 12.5
+    - initial_due [datetime.date, datetime.datetime or string]: The date of the first invoice. ex: scheduled=datetime(2020, 3, 10)
```

Comment:
- initial_due [datetime.date, datetime.datetime or string]: date of the first invoice. ex: scheduled=datetime(2020, 3, 10)